modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java

```diff
@@ -200,6 +200,13 @@ public class CodeGenMojo extends AbstractMojo {
     @Parameter(name = "packageName", property = "openapi.generator.maven.plugin.packageName")
     private String packageName;
 
+    /**
+     * The package to use for generated model objects/classes
+     */
+    @Parameter(name = "resourceFolder", property = "openapi.generator.maven.plugin.resourceFolder")
+    private String resourceFolder;
+
+
```

Comment:
I don't think there should be change to this file as resourceFolder is not a global option.

It's a generator option, which should be passed via `additionalProperties`

Reply:
Then I don‘t get it how this should work.

Reply:
can you please PM me via Slack to discuss this further?

Reply:
I just reverted it should be fine now

Reply:
LGTM